packages/web-api-scan-runner/src/runner/runner.spec.ts

```diff
@@ -145,6 +154,14 @@ describe(Runner, () => {
         MockDate.set(dateNow);
 
         reportGeneratorMock = Mock.ofType<ReportGenerator>();
+        serviceConfigurationMock = Mock.ofType(ServiceConfiguration);
+        notificationQueueMessageSenderMock = Mock.ofType(NotificationQueueMessageSender);
```

Comment:
change this to Mock.ofType(NotificationQueueMessageSender, MockBehavior.Strict)

Reply:
you should use strict mocks to service classes that interacts with external services like db, queues etc, to make sure we interact with this class only in expected scenarios